js/ui/shellDBus.js

```diff
@@ -16,6 +17,11 @@ const Main = imports.ui.main;
 const Screenshot = imports.ui.screenshot;
 const ViewSelector = imports.ui.viewSelector;
 
+/* Occurs when an application is installed, meaning added to the desktop's app
```

Comment:
Maybe just saying "Occurs when an application is added to the app grid" is enough?